torch/csrc/jit/passes/quantization.cpp

```diff
@@ -144,21 +144,21 @@ Node* createIntReprNode(Value* v, Node* n) {
 }
 
 // Insert Quant-Dequant node pattern for quantizable node outputs
-void addQuantDeQuantNodes(
+Node* addQuantDeQuantNodesFor(
     Value* v,
+    Node* insert_point,
     const std::tuple<std::string, float, int>& qparam,
     at::ScalarType t = at::ScalarType::Undefined) {
   TORCH_INTERNAL_ASSERT(v != nullptr);
-  Node* n = v->node();
-  Node* quant = createQuantNode(v, n);
-  Node* intrepr = createIntReprNode(v, n);
-  Node* dequant = createDeQuantNode(v, n);
+  WithInsertPoint ins(insert_point);
+  Node* quant = createQuantNode(v, insert_point);
```

Comment:
How about inlining these calls here? There is nothing much happening inside anyway - asserts are already there and the scope stuff can be removed if we use `WithCurrentScope` (it works similar to `WithInsertPoint` but sets the scope rather than the insert point).

If we do this and hoist `WithInsertPoint ins` out of this function to its callsite we would be able to get rid of the second argument (`Node* insert_point`).

Reply:
I recollect one more point. we have perchannelquantization coming up, so createQuantNode() will actually get expanded to take more argument and create node based on qscheme. i think it will be beneficial to retain these api as is instead of inlining

Reply:
Added the WithCurrentScope as suggested but retaining the createQuantNode() api due to reason stated above

Reply:
My concern about the current implementation is that it's misleading. For instance, a call `createQuantNode(v, insert_point)` looks like it would insert whatever it does into `insert_point`, while in fact it uses this argument for just retrieving the graph from it. My previous suggestions were one potential way to resolve it. You points are also valid, so I don't object to having these functions separate, but we still need to make them well-defined and easily understandable one way or another before it gets in.

Reply:
I can add a note and think of better name. if u have something in mind please suggest

Reply:
Done based on above comment